data/mods/classic_zombies/contagion/effect.json

```diff
@@ -0,0 +1,67 @@
+[  {
+    "//": "bitten by a zombie",
+    "type": "effect_type",
+    "id": "zombie_virus",
+    "name": [ "Zombie bite", "Zombie contagion", "Zombie contagion" ],
+    "desc": [
+      "You've got a painful bite from a zombie.  You know how this ends.",
+      "The area around your zombie bite is festering and red, and you're feeling hot and cold at once.",
+      "Everything's so dark.  You can feel it, you don't have much time left."
+    ],
+    "apply_message": "You've been bitten.  It's bad.",
+    "death_msg": "You succumb to your zombie bite.",
+    "apply_memorial_log": "Died of the zombie plague.",
+    "chance_kill": [ [ -1, 1 ], [ -1, 1 ], [ 1, 100000 ] ],
+    "death_event": "dies_of_infection",
+    "max_intensity": 5,
```

Comment:
Not too familiar with effects, but it looks like this only has 3 stages.

Reply:
woops, forgot to fix that one, thanks